src/documents/management/commands/manage_superuser.py

```diff
@@ -12,31 +12,42 @@ class Command(BaseCommand):
 
     help = """
         Creates a Django superuser based on env variables.
+        PAPERLESS_ADMIN_USER name of admin user (Default: admin)
+        PAPERLESS_ADMIN_MAIL admin email address (Default: root@localhost)
+        PAPERLESS_ADMIN_PASSWORD (NODEFAULT)
+
+        Logic:
+              Check if password is set, if not exit
+              Check is admin user exists, if exists exit
+                                          else create
     """.replace("    ", "")
 
     def handle(self, *args, **options):
+        # Get info from env
+        username = os.getenv('PAPERLESS_ADMIN_USER', 'admin')
+        mail = os.getenv('PAPERLESS_ADMIN_MAIL', 'root@localhost')
+        password = os.getenv('PAPERLESS_ADMIN_PASSWORD')
 
-        username = os.getenv('PAPERLESS_ADMIN_USER')
-        if not username:
+        # Return if email address does not pass basic validation
+        if not re.fullmatch(r"[^@]+@[^@]+", mail):
+            self.stdout.write(
+                'Given email address failed '
+                'validation.')
             return
 
-        mail = os.getenv('PAPERLESS_ADMIN_MAIL', 'root@localhost')
-        password = os.getenv('PAPERLESS_ADMIN_PASSWORD')
+        # Return if password is not set
+        if not password:
+            self.stdout.write(
+                'Make sure you specified "PAPERLESS_ADMIN_PASSWORD" in your '
+                '"docker-compose.env" file.')
+            return
 
         # Check if user exists already, leave as is if it does
-        if User.objects.filter(username=username).exists():
-            user: User = User.objects.get_by_natural_key(username)
-            user.set_password(password)
-            user.save()
-            self.stdout.write(f"Changed password of user {username}.")
-        elif password:
+        if not User.objects.filter(username=username).exists():
```

Comment:
AFAIK paperless is still a single user application. When changing the user PAPERLESS_ADMIN_USER name this could cause another user to be created. I'm not sure what would be the best way to handle this limitation. However, maybe this check should be changed to something like this?:
`User.objects.count() >= 0`

Reply:
Hi alex,
good observation, I had some thoughts about it but wanted it to be as simple as possible. 

We could check for super_user existing but from the persepctive that we do not want the setup to interact with a running system that's  not enough and your  "User.objects.count() >= 0" is the way to go.

I'll put more thoughts into it later and will adopt accordingly.

Regards

Schnuffle

Reply:
Added the two changes but I need to put some effort into adopting/enhancing the tests